Parse/PFConfig.m

```diff
@@ -104,8 +104,8 @@ - (BOOL)isEqual:(id)object {
         PFConfig *other = object;
 
         // Compare pointers first, to account for nil dictionary
-        return self.parametersDictionary == other.parametersDictionary ||
-            [self.parametersDictionary isEqual:other.parametersDictionary];
+        return (self.parametersDictionary == other.parametersDictionary ||
```

Comment:
This change is necessary, because it even conflicts with automatic indentation (without parenthesis - it's going to put the second line with the same indentation as `return`, which is confusing as hell. I am going to leave this one out.